examples/functions/functions_tester/test/integration/triggerMessage.test.js

```diff
@@ -0,0 +1,50 @@
+const { app_id } = require("../../realm_config.json");
+const Realm = require("realm");
+const { BSON } = require("realm");
+
+const app = new Realm.App(app_id);
+
+beforeAll(async () => {
+  await app.logIn(Realm.Credentials.anonymous());
+});
+
+afterAll(async () => {
+  await app.currentUser.logOut();
+});
+
+describe("trigger messenger", () => {
+  it("writes success to document", async () => {
+    // create doc should initiate trigger
+    const db = app.currentUser
+      .mongoClient("mongodb-atlas")
+      .db("triggerExample");
+    const collection = db.collection("messages");
+    const _id = BSON.ObjectId();
+    await collection.insertOne({
+      _id,
+      needsTriggerResponse: true,
+      result: null,
+    });
+
+    // Trigger should fire within the next 5 seconds
+    for (let attempt = 0; attempt < 10; ++attempt) {
```

Comment:
nit: standard to postfix ++ unless reason otherwise

```suggestion
    for (let attempt = 0; attempt < 10; attempt++) {
```

Reply:
That's not any standard I have ever heard of or would subscribe to. There is a semantic difference.

Reply:
Yeah, but it's always been the wrong way to do it. You don't need the reference to the value before the increment, so there's no reason to use postfix. Some people even argue that either increment operator should be abolished and one should only use `+=`. I don't get it.